packages/office-ui-fabric-react/src/components/Callout/Callout.scss

```diff
@@ -25,6 +25,7 @@ $Callout-smallbeak-slant-width: 16px;
   @include ms-drop-shadow(0,0, 5px, 0px);
   position: absolute;
   box-sizing: border-box;
+  border: 1px solid $ms-color-neutralLight;
```

Comment:
The border was originally removed in an update to the people picker's callout, so I didn't think it was a big deal to add it back in another update to the people picker's callout. You're right though, I'll pull it out here and start a new PR.